config.default.js

```diff
@@ -21,6 +21,8 @@ const config = {
     token: 'xoxb-Your-Token',
     autoReconnect: true
   },
+  disablePretext: true,
+  disableFootnote: true,
```

Comment:
Talk to me about why you'd want to disable the footer?

Reply:
It was a request from my director and the owner of the company I work at, they wanted to slim to payload as much as possible while still retaining some additional information in the minimal view.